plugins/Website.py

```diff
@@ -39,15 +40,29 @@ def init(self, logger):
         import re
         # From RFC 1738 paragraph 2.1
         self.HasScheme = re.compile(r"^[a-zA-Z0-9.+-]+://")
+        self.strippable_queryparameters = self._load_trackingparameter_list()
 
         self.errors[30931] = self.def_class(item = 3093, level = 2, tags = ['value', 'fix:chair'],
             title = T_('The URL contains a space'))
         self.errors[30932] = self.def_class(item = 3093, level = 3, tags = ['value', 'fix:chair'],
             title = T_('The URL does not have a valid scheme'))
+        self.errors[30933] = self.def_class(item = 3093, level = 3, tags = ['value', 'fix:chair'],
+            title = T_('Invalid URL'))
+        self.errors[30934] = self.def_class(item = 3093, level = 3, tags = ['value', 'fix:chair'],
+            title = T_('Tracking parameter in URL'),
+            fix = T_('Strip the tracking parameter from the URL. Verify that the URL still works afterwards.'),
+            resource = 'https://github.com/mpchadwick/tracking-query-params-registry')
 
     def _bad_url(self, tag, tags):
         return T_("Bad URL {k}={v}", k = tag, v = tags[tag])
 
+    def _load_trackingparameter_list(self):
+        # Permission to use via https://github.com/mpchadwick/tracking-query-params-registry/issues/15
+        url = "https://raw.githubusercontent.com/mpchadwick/tracking-query-params-registry/master/_data/params.csv"
+        csvlines = urlread(url, 30).split("\n")[1:] # Discard first line, it contains headers
+        # Filter entries of len<=2 for safety. First column contains the parameter
+        return set(filter(lambda p: len(p) > 2, map(lambda line: line.split(",", 1)[0].strip(), csvlines)))
```

Comment:
Use the python CSV parser rather than manual parser

Reply:
Done